ts/menu.ts

```diff
@@ -15,41 +15,34 @@
  *  limitations under the License.
  */
 
-
 /**
- * @fileoverview Interface specification for context menus.
- *
+ * @file Interface specification for context menus.
  * @author volker.sorge@gmail.com (Volker Sorge)
  */
 
-import {Item} from './item.js';
-import {Postable} from './postable.js';
-import {VariablePool} from './variable_pool.js';
-import {KeyNavigatable} from './key_navigatable.js';
-import {MouseNavigatable} from './mouse_navigatable.js';
+import { Item } from './item.js';
+import { Postable } from './postable.js';
+import { VariablePool } from './variable_pool.js';
+import { KeyNavigatable } from './key_navigatable.js';
+import { MouseNavigatable } from './mouse_navigatable.js';
 
 export interface Menu extends Postable, KeyNavigatable, MouseNavigatable {
-
   /**
    * The base menu structure this menu belongs to.
-   * @type {Menu}
    */
   baseMenu: Menu;
 
   /**
    * The items in that menu.
-   * @type {Item[]}
    */
   items: Item[];
 
   /**
-   * @type {VariablePool<string | boolean>} The variable pool associated to
    *     this menu.
```

Comment:
You may need to remove this line by hand.

Reply:
Thanks for catching that. I was quite carefully comparing diffs after running my `sed` scripts, but must have missed this one.